fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -70,6 +70,31 @@ get_jar_file() {
   fi
 }
 
+ceiling() {
+  awk -vnumber="$1" -vdiv="$2" '
+    function ceiling(x){
+      return x%1 ? int(x)+1 : x
+    }
+    BEGIN{
+      print ceiling(number/div)
+    }
+  '
+}
+
+# Based on the cgroup limits, figure out the max number of core we should utilize
+max_cores() {
```

Comment:
well, yeah. maybe I should rename it to `container-options` since it supposed to deal with container peculiarities. It the split might be arbitrary. Let me rework it a bit as we could expose the max available container memory the same way as environment variable (and then use this env var to set the java options).